src/arrayMethodSort.js

```diff
@@ -4,8 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compareAsStrings = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA > stringB) {
+      return 1;
+    }
+
+    if (stringA === stringB) {
+      return 0;
+    } else {
+      return -1;
+    }
```

Comment:
There is no need to check if strings are equal. 
You can do this in simplest way:
```
if (stringA > stringB) {
  return 1;
}

return -1;
```